apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -0,0 +1,210 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+
+/* global bridge, Utils */
+
+/* exported MozMobileMessageShim */
+
+'use strict';
+(function(exports) {
+
+/**
```

Comment:
nit: I see we use different approach to indent code inside "module" (either with usual 2 spaces indent or without indent at all like here). Let's stick to a single way :) What do you prefer @steveck-chung, @julienw ?